
#### Scripts
##### SearchIncidentsV2
- Fixed an issue where the value of an incident id from type int raised a TypeError exception.
- Updated the Docker image to demisto/python3:3.9.2.17957.